components/wasm-opt-sys/build.rs

```diff
@@ -36,14 +36,17 @@ fn main() -> anyhow::Result<()> {
     // include from these same dirs.
     CFG.exported_header_dirs.push(&src_dir);
     CFG.exported_header_dirs.push(&tools_dir);
-    CFG.exported_header_dirs.push(&output_dir);
+    CFG.exported_header_dirs.push(output_dir);
 
     #[cfg(feature = "dwarf")]
     {
         let llvm_include = llvm_dir.join("include");
         CFG.exported_header_dirs.push(&llvm_include);
     }
 
+    let fp16_dir = binaryen_dir.join("third_party/FP16/include");
+    CFG.exported_header_dirs.push(&fp16_dir);
```

Comment:
This is needed to compile. Not sure if I should move this up a few lines so it's next to where src, tools, and output are added to the exports.